code/artificial_intelligence/src/minimax/minimax.py

```diff
@@ -0,0 +1,37 @@
+from collections import namedtuple
+import random
+
+#from utils import argmax
+
+infinity = float('inf')
+GameState = namedtuple('GameState', 'to_move, utility, board, moves')
+
+# ______________________________________________________________________________
+# Minimax Search
+
+
+def minimax_decision(state, game):
```

Comment:
`game` is a user-defined datatype. You must define it and write complete code that makes the usage of the method (minimax in this case) clear.